tests/runtests.sh

```diff
@@ -207,13 +222,24 @@ ${SCRIPT_DIR}/smoketests.py --toaster_url="$toastername:8000" \
                 $timeout_arg \
                 $pokybranch_arg
 echo "smoketests PASSED!"
-${SCRIPT_DIR}/checkartifacts.sh $tempdir/toasterbuild/build-toaster-2
+
+${SCRIPT_DIR}/checkartifacts.sh $tempdir/$toasterbuild/build-toaster-2
 echo "checkartifacts PASSED!"
 
-grep -q "No migrations to apply" $toasterlog
-echo "no migration PASSED"
+grep -q "No migrations to apply" $toasterlog || test "" != $LOCALDIR
+echo "no migration priming PASSED"
 
+if grep -q "INFO Fetching layers" $toasterlog; then
+    if [ "" == $LOCALDIR ]; then
+	exit 1
```

Comment:
i like trapping exit too, so it always cleans up.
-b

On Sun, Aug 21, 2016 at 5:24 PM, Randy Witt <notifications@github.com>
wrote:

> In tests/runtests.sh
> <https://github.com/crops/toaster-container/pull/8#discussion_r75606914>:
>
> >
> > +if grep -q "INFO Fetching layers" $toasterlog; then
> > +    if [ "" == $LOCALDIR ]; then
> > +	exit 1
>
> I think the error handling needs to also trap exit, or this should be
> changed to be "false" instead of "exit 1".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/crops/toaster-container/pull/8/files/9dda39d3cf58b81c6820ef1528d2728a962f26a1..ce007d22f3af1722580467f78fb7d1d616c6e320#r75606914>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACkxjdNvdrmc1Vc4v9fT4cNPPTllHPKBks5qiOxVgaJpZM4Jo75K>
> .
>